vision_msgs/BoundingBox2D bbox
string label
float64 confidence   